src/coprocessor/select/xeval/builtin_math.rs

```diff
@@ -15,7 +15,7 @@ use tipb::expression::Expr;
 use coprocessor::codec::datum::Datum;
 use super::{Evaluator, EvalContext, Result, Error};
 
-const ERROR_UNIMPLEMENTED: &'static str = "unimplemented";
+const _ERROR_UNIMPLEMENTED: &'static str = "unimplemented";
```

Comment:
Because it did not use and would result in compile error. 

Maybe it can be deleted.